mix.lock

```diff
@@ -45,6 +45,7 @@
   "sext": {:git, "https://github.com/uwiger/sext.git", "615eebcf975ec4b4561c6f2b2bc433dabdb2be0f", [tag: "1.4.1"]},
   "sha3": {:git, "https://github.com/szktty/erlang-sha3.git", "dbdfd12ace2fc7e530e697fef81e1e72e90740f9", [ref: "dbdfd12"]},
   "ssl_verify_fun": {:hex, :ssl_verify_fun, "1.1.1", "28a4d65b7f59893bc2c7de786dec1e1555bd742d336043fe644ae956c3497fbe", [:make, :rebar], [], "hexpm"},
+  "temp": {:hex, :temp, "0.4.5", "c62ee92c778ddebb2738212fecea372f1a9beba10bacbd7e54e7efe29357a82e", [:mix], [], "hexpm"},
```

Comment:
why is this added in the lockfile, but not mixfile?

Reply:
Unused library, will remove it